OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -252,7 +252,7 @@ protected function httpRequest($url, $content = null, $headers = array(), $metho
         try {
             $this->httpClient->send($request, $response);
         } catch (ClientException $e) {
-            throw new HttpTransportException('Error while sending HTTP request', $this->getName(), $e->getCode(), $e);
+            // Do nothing: user will be redirected to failure_path
```

Comment:
Fixes issue: Google API errors generate CRITICAL error.

We just keep going on without useful user, and the error happens later and is handled appropriately.